robyn/__init__.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, file_object: str, config: Config = Config()) -> None:
         load_vars(project_root=directory_path)
         logging.basicConfig(level=self.config.log_level)
 
-        if self.config.log_level.lower() != "warn":
+        if self.config.verbose is True:
             logger.info(
                 "SERVER IS RUNNING IN VERBOSE/DEBUG MODE. Set --log-level to WARN to run in production mode.",
```

Comment:
I think this message is a bit out of place after changing the if statement.

Reply:
I don't understand what do you mean here. Should I remove the message, the if statement?

Reply:
Sorry, I probably wasn't clear! Previously the message would appear when the log_level is not `WARN`, and the message says to set the log-level to `WARN` on production, so on production it wouldn't appear. After the change, even if the log-level is `WARN` - but verbose is truthy, then the message would appear. Which then doesn't make sense.

Since `--verbose` is now very... well... verbose 😅  I feel that the message is redundant. If someone runs on production with `--verbose` flag, they probably do it purposefully. WDYT?